src/stickyfill.js

```diff
@@ -495,6 +511,6 @@ if (!seppuku) init();
 if (typeof module != 'undefined' && module.exports) {
     module.exports = Stickyfill;
 }
-else {
+else if (isWindowDefined) {
```

Comment:
With `isWindowDefined` flag removed `window.Stickyfill = Stickyfill;` can be changed to `this.Stickyfill = Stickyfill;`.

Reply:
actually, I get an error that `this` is undefined if I do that